app/routes/review_item/supervisor_messages.js

```diff
@@ -1,12 +1,14 @@
-import MessagesBaseRoute from 'shared-goodcity/routes/messages_base';
+import MessagesBaseRoute from "shared-goodcity/routes/messages_base";
 
 export default MessagesBaseRoute.extend({
   renderTemplate() {
-    this.render('message_template', {controller: 'review_item.supervisor_messages'});
+    this.render("message_template", {
+      controller: "review_item.supervisor_messages"
+    });
   },
 
   afterModel() {
-    var itemId = this.modelFor('review_item').get('id');
-    this.store.query('version', { item_id: itemId, for_item: true });
+    var itemId = this.modelFor("review_item").item.get("id");
```

Comment:
```suggestion
    const itemId = this.modelFor("review_item").get("item.id");
```

Also:
1. why are we fetching versions ?
2. why are we fetching versions _without_ awaiting the response.  Looks odd to me

I know you didn't write this code, but we should double check. This looks dodgy